test/gui/shared/steps/steps.py

```diff
@@ -1466,3 +1462,50 @@ def step(context, username, foldername):
 @Then("credentials wizard should be visible")
 def step(context):
     waitForObject(AccountConnectionWizard.CREDENTIAL_PAGE)
+
+
+@When("user selects configure_synchronization_manually option in advanced section")
+def step(context):
+    clickButton(
+        waitForObject(
+            names.syncModeGroupBox_configureSyncManuallyRadioButton_QRadioButton
```

Comment:
I think we can move this variable to `AccountConnectionWizard` page object.
Let's try to use the page object as far as we can.